### PR TITLE
Landing page as / route and onboarding moved to /onboarding

### PR DESCRIPTION
**Pull request by matviikotolyk**
## Description

Landing page set as the default (/) route and the onboarding page moved to the /onboarding route. After signing up/in the user is redirected to the onboarding page instead of back to the landing page. The youtube demo is added to the page.

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Tests (adding or updating tests)
- [ ] Documentation update

## Testing Instructions

Check out the `matviikotolyk/landing-page-default-route` branch. Try signing in from the landing page and checking if it redirects to the onboarding page. Look out for any inaccurate redirects

## Screenshots (if applicable)
Screenshot 2024-12-20 at 14 29 42

### CODE REVIEW

File: src/app/page.tsx
Comment (avalucianelson) [inline src/app/page.tsx:1]: I believe Kyron had cautioned against using "use client"; -- could be wrong and don't think it's a big deal!

**Review by avalucianelson — COMMENTED** [review]
As far as I can tell, this PR looks good. Saw you tagged Uce to review so will leave it to him to approve. As far as I can tell there are no issues but another set of eyes would be beneficial.

File: src/app/page.tsx
Comment (matviikotolyk) [reply src/app/page.tsx:1]: Thanks for bringing this up! I think there it's fine since there are no dynamic componetns

**Comment by matviikotolyk** [comment]
@avalucianelson feel free to also approve if you can; I think Uce was assigned through the random assigned job

**Review by avalucianelson — APPROVED** [review]
This all looks great, Matvii! Thank you so much for your contribution. I love incorporating the video demo into the landing page - great idea.

**Pull request merged**
